navigation/src/main/java/com/simtop/navigation/BeerListNavigation.kt

```diff
@@ -1,13 +1,15 @@
-package com.simtop.feature.beerslist.navigation
+package com.simtop.navigation
 
+import androidx.annotation.Keep
+import androidx.navigation.NavController
 import androidx.navigation.dynamicfeatures.DynamicExtras
 import com.simtop.beerdomain.domain.models.Beer
-import com.simtop.feature.beerslist.BeersListFragment
 
+//@Keep
 interface BeerListNavigation {
     fun fromBeersListToBeerDetail(
         beer: Beer,
-        fragment: BeersListFragment,
+        navController: NavController,
```

Comment:
I had to make changes and move to a separate module because KSP works differently from KAPT,  because KSP doesn't generate things in parallel like KAPT and the unifies everything in the end.